andela/ah-frontend-prime
# #165819922 Implement Landing Page

#### What does this PR do?
Implements landing page for Authors Haven

#### Description of Task to be completed?
- Edit the landing page to contain buttons for the login and signup
- Add articles for the backend api
- Style the landing page according to the mockups

#### How can this be tested?
Install all dependencies

- `$ npm install`

Start the server

- `$ npm start`

Checkout the landing page that appears  in your browser and click the Sign in button in the top right corner.

#### Screen shots:

Landing Page

<img width="1438" alt="Screenshot 2019-05-10 at 08 21 22" src="https://user-images.githubusercontent.com/43245704/57504355-c5f4f180-72fc-11e9-87cf-6349e859851d.png">



#### Relevant Pivotal Tracker Stories:
[165819922](https://www.pivotaltracker.com/story/show/165819922)
